packages/react/src/playground/sections/NotificationsSection.js

```diff
@@ -1,6 +1,12 @@
 import React, { PureComponent } from "react";
 import PlaygroundSection from "../PlaygroundSection";
-import { Button, Notifications, Notification, TextLink } from "../../hig-react";
+import {
+  Button,
+  Notifications,
+  NotificationV1,
```

Comment:
I feel a little queasy about the `V1`, but I see why you did it and I can't think of a better solution without changing our packaging versioning strategy. This is our first multi-version component. I'll add another story to move this into the future structure.

Reply:
In the mean time would you change this line to `NotificationV1 as Notification`?

Reply:
sounds good.